src/pages/dev/amplifier/chain-integration/relay-messages.mdx

```diff
@@ -0,0 +1,157 @@
+# Relay messages with Amplifier
+
+import { Callout } from "/src/components/callout"
+
+Chains integrated with Amplifier can [pass GMP messages](https://docs.axelar.dev/dev/general-message-passing/overview) to one another through the following flow:
+
+1. When a user sends a message to the chain, a `callContract` event is emitted.
+1. The relayer picks up the event and calls [`verify_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L9) on the chain’s [Axelar (internal) gateway contract](https://github.com/axelarnetwork/axelar-amplifier/blob/main/doc/src/contracts/gateway.md).
+1. For chains that rely on verifier votes, the Axelar gateway calls [`verify_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L9) on the voting verifier, which prompts verifiers to begin voting.
```

Comment:
Feels like you are talking about verify_messages on the voting verifier, shouldn't we link to that code instead? This link is for the gateway.